test/e2e/test_setting.py

```diff
@@ -196,6 +219,14 @@ def test_create_item_after_spares_definition_set(self):
     def test_create_item_after_spares_definition_set_with_multiple_types(self):
         """Test creating an item after the spares definition is set to one with multiple system type IDs in it.
         (This test is not repeated for update/delete as the logic used is the same)"""
+        # Add a custom rule to allow items to be created in Operational systems
+        custom_creation_rule_id = self.add_rule(
+            {
+                "src_system_type_id": None,
+                "dst_system_type_id": SYSTEM_TYPE_OUT_DATA_OPERATIONAL["_id"],
+                "dst_usage_status_id": USAGE_STATUS_OUT_DATA_NEW["_id"],
+            }
+        )
```

Comment:
```suggestion
        custom_creation_rule_id = self.add_rule(RULE_DOCUMENT_DATA_OPERATIONAL_CREATION)
```
Would mean it can be reused below if its added to the mock_data file, name matches that from the function its used in.